### PR TITLE
Documentation/fastq

### PR DESCRIPTION
**Pull request by wow-such-code**
Move documentation from repo front page to fastq dropbox, change formatting

### CODE REVIEW

**Comment by wow-such-code** [comment]
[register-fastq ETL readme](https://trello.com/c/YlXKaq8b/540-register-fastq-etl-readme)

File: drop-boxes/register-fastq-dropbox/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#QBiC-register-fastq-dropbox
+
+##Resulting data model in openBIS
+Q_TEST_SAMPLE -> Q_NGS_RAW_DATA (with sample code) -> DataSet (directory with files contained)
```
Comment (jenniferboedker) [inline drop-boxes/register-fastq-dropbox/README.md:4]: What do you want to indicate here? Do you want to indicate which files to use?
Is it possible to add a diagram to indicate this and textually describe what you want to show here?

Comment (jenniferboedker) [reply drop-boxes/register-fastq-dropbox/README.md:4]: Also here maybe add example OpenBis codes. I think they must end with `_NGS` ?

Comment (jenniferboedker) [reply drop-boxes/register-fastq-dropbox/README.md:4]: Ah wait. You are registering the data to the sample. then maybe the code for the sample at the correct level is better

File: drop-boxes/register-fastq-dropbox/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+##Description
+For paired-end sequencing reads in FASTQ format, the file structure needs to look like this:
+
+```
+<QBIC sample code>.fastq // Directory
+    |-- <QBIC sample code>_R1.fastq
+    |-- <QBIC sample code>_R1.fastq.sha256sum
+    |-- <QBIC sample code>_R2.fastq
+    |-- <QBIC sample code>_R2.fastq.sha256sum
+```
```
Comment (jenniferboedker) [inline drop-boxes/register-fastq-dropbox/README.md:15]: How can I verify this structure from the code?

File: drop-boxes/register-fastq-dropbox/README.md (hunk 1 above)
Comment (wow-such-code) [reply drop-boxes/register-fastq-dropbox/README.md:4]: Yes, this is just how the entities are connected in openBIS. I don't know if this is necessary for the user in the end, but it was like that for the previous dropbox/etl script.

File: drop-boxes/register-fastq-dropbox/README.md (hunk 2 above)
Comment (wow-such-code) [reply drop-boxes/register-fastq-dropbox/README.md:15]: https://github.com/qbicsoftware/etl-scripts/blob/master/drop-boxes/register-fastq-dropbox/register-fastq-dropbox.py#L123 ff are probably best.
The folder is registered, contained files which contain ".testorig" and ".origlabfilename" are removed (nothing is done with this metadata, but it is how it is at the moment) and everything else is registered. However, as you can see the .sha256sums are not necessary for the script to work. It's just that they are automatically created by the datahandler. and are thus part of the directory. Shall I remove that part, as it's a specific implementation for QBiC that does not have anything to do with the ETL script itself?

Comment (jenniferboedker) [reply drop-boxes/register-fastq-dropbox/README.md:15]: But how come R1 and R2 into play? Does it matter for the script (didn't find a line where this was required)? Or should it just display that both strands should be provided?

Comment (wow-such-code) [reply drop-boxes/register-fastq-dropbox/README.md:15]: The latter, the script does not care :)

**Comment by jenniferboedker** [comment]
after you rebased to master and moved the documentation to the main readme I will approve :)

**Comment by wow-such-code** [comment]
We will close this as the documentation currently found in the main readme is sufficient.

**Pull request closed without merging**